velox/core/PlanNode.h

```diff
@@ -659,12 +659,16 @@ using PartitionFunctionFactory =
 /// different from input.
 class LocalPartitionNode : public PlanNode {
  public:
+  enum class Type { kGather, kRepartition, kReplicate };
```

Comment:
Please can you add a comment what each of these mean ?

Also, the naming of "Type" was a bit misleading... maybe "strategy" is more clear ?